### PR TITLE
Invitation manager

### PR DESCRIPTION
**Pull request by RabiaSajjad**

### CODE REVIEW

File: modules/custom/invitation_manager/invitation_manager.module [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * This function loads assets from the Service Canada GC Invitation Manager library
+ * Only public page sessions (not when a user is logged in) load survey pop-up
+ */
+function invitation_manager_page_attachments(array &$attachments) {
+  $node = \Drupal::request()->get('node');
+  if (is_object($node) && 
+    $node->getType() === 'landing_page' && 
+    \Drupal::currentUser()->isAuthenticated() === false) {
+
+    if (!in_array('core/jquery', $attachments['#attached']['library']))
```
Comment (JVickery-TBS) [inline modules/custom/invitation_manager/invitation_manager.module:19]: @RabiaSajjad Is this check for the jquery needed if `core/jquery` is already stated as a dependency in the libraries.yml?

**Review by JVickery-TBS — COMMENTED** [review]
@RabiaSajjad Yeah this all looks good. I made the one comment question the extra if statement for the check for the `core/jquery` library.

File: modules/custom/invitation_manager/invitation_manager.module (hunk 1 above)
Comment (RabiaSajjad) [reply modules/custom/invitation_manager/invitation_manager.module:19]: thanks for reviewing @JVickery-TBS. The invitation manager code needs jQuery to execute, so this is only to make sure that jQuery is loaded before invitation manger script starts executing

**Comment by JVickery-TBS** [comment]
@RabiaSajjad This looks good to go then!

**Pull request merged**